CRM/Core/DAO/CustomField.php

```diff
@@ -812,6 +821,21 @@ public static function &fields() {
           'localizable' => 0,
           'add' => '4.5',
         ],
+        'fk_entity' => [
```

Comment:
I think I would have called this reference_entity - but non-blocking. 

FK is not too bad in the scheme of Civi abbreviations in that we use it a lot so it's not quite the guess the meaning game - but at the same time when I looked at it I tried to find the Foreign Key and didn't find one, so I looked at the description & it explained that it is a reference_entity

Reply:
APIv4 `getFields` outputs a property called `fk_entity`, so this matches that.
I did ponder naming this something else, but the lack of matching seems to undercut any benefits of doing so.